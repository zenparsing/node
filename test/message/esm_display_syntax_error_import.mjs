// Flags:  --module
/* eslint-disable no-unused-vars */
/* eslint-disable node-core/required-modules */
import '../common/index.mjs';
import {
  foo,
  notfound
} from '../fixtures/es-module-loaders/module-named-exports.mjs';
