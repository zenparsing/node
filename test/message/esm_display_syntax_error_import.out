file:///*/test/message/esm_display_syntax_error_import.mjs:7
  notfound
  ^^^^^^^^
SyntaxError: The requested module '../fixtures/es-module-loaders/module-named-exports.mjs' does not provide an export named 'notfound'
    at ModuleLoader._instantiateModule (internal/modules/esm/module_loader.js:*:*)
    at ModuleLoader._instantiateAndEvaluate (internal/modules/esm/module_loader.js:*:*)
    at async ModuleLoader.importModule (internal/modules/esm/module_loader.js:*:*)
